src/dotnet/CarbonAware.DataSources.WattTime/test/WattTimeDataSourceTests.cs

```diff
@@ -79,7 +79,7 @@ public async Task GetCarbonIntensity_ReturnsResultsWhenRecordsFound()
         Assert.AreEqual(balancingAuthority.Abbreviation, first.Location);
         Assert.AreEqual(startDate.DateTime, first.Time);
 
-        this.LocationSource.Verify(r => r.ToGeopositionLocation(location));
+        this.LocationSource.Verify(r => r.ToGeopositionLocationAsync(location));
```

Comment:
I think this should be an awaitable call right?

Reply:
I dont think we need await since this just verifies that the method is called